metaflow/environment.py

```diff
@@ -99,7 +99,7 @@ def get_boto3_copy_command(self, s3_path, local_path, command="download_file"):
         )
 
     def get_package_commands(self, code_package_url, pip_install=True):
-        cmds = ["set -ex",  # -x for debugging, it prints out the commands executed
+        cmds = ["set -e",
```

Comment:
Do we want to re-enable this for debugging?

Reply:
added check for `debug.subcommand`